qiita_db/util.py

```diff
@@ -1331,9 +1331,20 @@ def generate_study_list(user, visibility):
                 WHERE study_id=qiita.study.study_id)
                 AS number_samples_collected]
     - all the BIOM artifact_ids sorted by artifact_id that belong to the study,
-      including their software deprecated value
-            (SELECT array_agg(row_to_json((artifact_id, qs.deprecated), true)
-                              ORDER BY artifact_id)
+      including their software deprecated value and their prep info file data
+      type values
+            (SELECT array_agg(row_to_json(
+                (artifact_id, qs.deprecated, (
+                    SELECT array_agg(DISTINCT data_type)
+                    FROM qiita.prep_template
+                    LEFT JOIN qiita.data_type USING (data_type_id)
+                    WHERE artifact_id IN (
+                        SELECT *
```

Comment:
It might be better to explicitly define what we want out of this table, rather than using wildcard (*), but then our tables don't exactly change often.